scanner/updater/export.go

```diff
@@ -18,19 +18,19 @@ import (
 	"github.com/stackrox/rox/scanner/updater/rhel"
 	"golang.org/x/time/rate"
 
-	// default updaters
+	// Default updaters. This is required to ensure updater factories are set properly.
 	_ "github.com/quay/claircore/updater/defaults"
 )
 
 // Export is responsible for triggering the updaters to download Common Vulnerabilities and Exposures (CVEs) data
-// and then outputting the result as a zstd-compressed file with .ztd extension
+// and then outputting the result as a zstd-compressed file named vulns.json.zst.
```

Comment:
Nitpick: Two places naming the file.

```suggestion
// and then outputting the result as a zstd-compressed file.
```

Reply:
Yes, but in two places, so updating one will require updating the other. But fine to go.